templates/default/consul/hiveserver2-health.sh.erb

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -e
+
+curl --fail --silent http://$HOSTNAME:<%= node['hive2']['hs2']['metrics_port'] %> > /dev/null 2>&1
```

Comment:
Why do we need 2 files for this? The only difference is the port, can you change them to use a variable instead?

Reply:
Changed to one template file